javascript/react/intro-react/what-i-need-to-use-react.md

```diff
@@ -42,7 +42,7 @@ Although not required, adopting a modern **build pipeline** is highly recommende
 - a **bundler**[1] such as `webpack` or `browserify` to improve load time by modularising your code
 - a **compiler** such as `Babel`[2]
 
-Using `Babel` compiler will allow you to write your application with both `ES6`[3] and `JSX` syntax. For this, you need to install `babel-preset-react` and `babel-preset-es2015` and enable them in your `.babelrc` config file.
+Using the `Babel` compiler will allow you to write your application with both `ES6`[3] and `JSX` syntax. For this, you need to install `babel-preset-react` and `babel-preset-es2015` and enable them in your `.babelrc` config file.
```

Comment:
Replace ES6 with ESNext (and the corresponding footnote).

[More about ESNext](https://en.wikipedia.org/wiki/ECMAScript#ES.Next)

Reply:
Added the following footnote:

`ES.Next` is a dynamic name that represents whatever the next version of ECMAScript is at the time of writing.